recipes/spefile/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.6" %}
+
+package:
+    name: spefile
+    version: {{ version }}
+
+source:
+    path: src
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - numpy
+
+test:
+    # Python imports
+    imports:
+        - spefile
+
+about:
+    home: https://pythonhosted.org/pyspec/
```

Comment:
Just add a comment about that here in the recipe. I am sure you and your group are aware of that, but others like me will need the comment :grimacing: